src/main/java/matvey/russkov/lecture3/StackTest.java

```diff
@@ -0,0 +1,25 @@
+public class StackTest {
+    public static void fill(Stack<String> stack) {
+        for (int i = 0; i < 5; i++) {
+            stack.push("e" + i);
+        }
+    }
+    public static <E> void dump(Stack<E> stack) {
+        System.out.println(stack.size());
+        while (stack.size() != 0) {
```

Comment:
You should get used to use methods like isEmpty rather than test if the size == 0 wherever it possible